app/models/user.rb

```diff
@@ -34,8 +34,8 @@ class User < ActiveRecord::Base
       original: '300x300>'
     },
     s3_storage_class: :reduced_redundancy,
-    s3_permissions: :private,
-    default_url: '/data/profile_picture_default.png'
+    s3_permissions: :public_read,
```

Comment:
You might need to update existing permissions on the image files already stored on our S3.